dataserver.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const arsenal = require('arsenal');
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger').logger;
+
+if (config.backends.data === 'file') {
+    /**
+     * Configure the file paths for data if using the file
+     * backend. If no path provided, uses data at the root of
+     * the S3 project directory.
+     */
+    const dataPath = process.env.S3DATAPATH ?
```

Comment:
Now that I think about it, in order to put this back in Config.js requires mdserver.js to parse the whole config as S3 does, because we can and some will run the dmd as a process independent of S3, which means that the dmd would have to have a valid S3 config. I don't like this approach too much, I think they should stay independent. (the `config.json` of the dmd side should ideally be able to contain only what is required to launch the dmd).

Another approach is to let the S3 server pass on the data path configuration bit to the dmd initialization code, but here the issue is that it should not be S3 server's responsiblity to choose the path to store data to but the dmd, which again may be on a remote machine.

Let's just make sure that `S3DATAPATH` and `S3METADATAPATH` are documented clearly if not already the case.